features/users/logging_out.feature

```diff
@@ -5,9 +5,25 @@ Feature: User Logging out
   Scenario: Logging out successfully
     Given a configuration of:
     """
-      ActiveAdmin.register Post
```

Comment:
Last thing I noticed from looking at the final diff is that this setup is unnecessary. I opened #5584 to clean it up in all cucumber scenarios related to user sessions. Once we merged that PR we can rebase this one, and get it in! :+1:

Reply:
It's merged now! Can you rebase this PR and squash it to a single commit?

Reply:
done 🙂

Reply:
Great thanks!